Ginger/GingerCoreNET/RunLib/CLILib/CLIHelper.cs

```diff
@@ -236,32 +236,37 @@ private void SelectRunset()
         }
 
         private void SelectEnv()
-        {            
+        {
             Reporter.ToLog(eLogLevel.DEBUG, "Selected Environment: '" + Env + "'");
-            ProjEnvironment env = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<ProjEnvironment>().Where(x => x.Name.ToLower().Trim() == Env.ToLower().Trim()).FirstOrDefault();
-            if (env != null)
-            {
-                mRunsetExecutor.RunsetExecutionEnvironment = env;
-            }
-            else
+            if (String.IsNullOrEmpty(Env) == false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)